serverless-functions/src/functions/features/chat-transfer/flex/create-transfer-task.js

```diff
@@ -76,7 +74,7 @@ exports.handler = prepareFlexFunction(requiredParameters, async (context, event,
     // the customer front end can trigger cancelling tasks associated to the chat channel
     // this is not critical to transfer but is ideal
     try {
-      if (createTaskSuccess)
+      if (result.success)
         await ChatOperations.addTaskToChannel({
           context,
           taskSid: newTask.sid,
```

Comment:
`newTask` no longer exists, instead `result.task.<whatever>` for both of these I believe.